src/scan.c

```diff
@@ -38,16 +39,17 @@ typedef struct {
 
 typedef A(cond_state_t) cond_stack_t;
 
-static const char     *file_start;
-static size_t          file_sz;
-static const char     *read_ptr;
-static hdl_kind_t      src_kind;
-static loc_file_ref_t  file_ref = FILE_INVALID;
-static int             colno;
-static int             lineno;
-static int             lookahead;
-static int             pperrors;
-static cond_stack_t    cond_stack;
```

Comment:
Might as well leave the indentation as it was for the unchanged lines as it can still line up.